From 98d4af769b3c1ee42240c7dfb3799a2ce5edb3f3 Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@remirepo.net>
Date: Tue, 26 Jun 2018 14:56:03 +0200
Subject: [PATCH] fix ce->name is a zend_string

---
 msgpack_convert.c | 2 +-
 msgpack_unpack.c  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/msgpack_convert.c b/msgpack_convert.c
index 117727e..85d1da6 100644
--- a/msgpack_convert.c
+++ b/msgpack_convert.c
@@ -314,7 +314,7 @@ int msgpack_convert_object(zval *return_value, zval *tpl, zval *value) /* {{{ */
         if (zend_call_function(&fci, &fcc) == FAILURE) {
             MSGPACK_WARNING(
                 "[msgpack] (%s) Invocation of %s's constructor failed",
-                __FUNCTION__, ce->name);
+                __FUNCTION__, ZSTR_VAL(ce->name));
 
             return FAILURE;
         }
diff --git a/msgpack_unpack.c b/msgpack_unpack.c
index 263341b..6c3f30b 100644
--- a/msgpack_unpack.c
+++ b/msgpack_unpack.c
@@ -557,7 +557,7 @@ int msgpack_unserialize_map_item(msgpack_unserialize_data *unpack, zval **contai
                     if (ce->unserialize == NULL) {
                         MSGPACK_WARNING(
                             "[msgpack] (%s) Class %s has no unserializer",
-                            __FUNCTION__, ce->name);
+                            __FUNCTION__, ZSTR_VAL(ce->name));
 
                         MSGPACK_UNSERIALIZE_FINISH_MAP_ITEM(unpack, key, val);
 
